messages/run-dfa.md

```diff
@@ -49,11 +49,11 @@ Specifies Java Virtual Machine arguments to override system defaults while execu
 
 # flags.targetSummary
 
-return location of source code
+source code location
 
 # flags.targetDescription
 
-Returns the source code location. Use glob patterns or specify individual methods with #-syntax. Multiple values are specified as a comma-separated list.
+Source code location. May use glob patterns. May specify individual methods with #-syntax. Specify multiple values as a comma-separated list.
```

Comment:
We should really re-review all of these summaries and descriptions at some point. This one was bugging me because of the word "Returns..." which doesn't make sense. I put this more inline with the "run" command's target flag summary and description.